### PR TITLE
Changed source of default value to the same as used in angular-translate

### PR DESCRIPTION
**Pull request by pfrankov**
I'm not sure that this is the best solution, but it works.

### CODE REVIEW

**Comment by DatenMetzgerX** [comment]
Hy Pavel

Thank you for your contribution. 

I'm not sure if this is a good idea. What you try to do is that you can use translate-default to specify the default text for a translated attribute, e.g. for the following snipped

```html
<a translate-attr-title="CLICK_TOOLTIP" translate-default="Click Me">...</a>
````

As i understand the specification, `translate-default-attr-title` should be used to specify the default text for the attribute translation: 

```html
<a translate translate-attr-title="CLICK_TOOLTIP" translate-attr-title-default="Click Me">...</a>
````

Or is the behavior explicitly stated in the specification? Because only because it behaves this way doesn't mean that it is intentionally... And personally I think that it is confusing for the following case (that is not correctly covered by your implementation, your implementation always uses `Click me` instead of the specific default text):

```html
<input translate-attr-title="CLICK_TOOLTIP" translate-default="Click Me" translate-attr-placeholder="NUMBER" translate-default-attr-placeholder="Number">...</a>
````

Anyway... Please create a unit test that covers these two cases and fix your implementation. If that is done, I can integrate it, if still needed.

**Comment by pfrankov** [comment]
According to [Angular Translate API docs](https://angular-translate.github.io/docs/#/api/pascalprecht.translate.directive:translate) there should be way to define default value for translation key.
```html
<translate translate-default="Incorrect email">INCORRECT_EMAIL</translate>
```
With my fix it should work as expected.

**Comment by DatenMetzgerX** [comment]
This should already be supported according to the following [test case](https://github.com/DatenMetzgerX/webpack-angular-translate/blob/master/test/cases/defaultText.html) defined in the following [spec](https://github.com/DatenMetzgerX/webpack-angular-translate/blob/master/test/plugin.specs.js#L131).

The line you have changed is for translated attributes. In this case, translate-default-title="test" needs to be used and not translate-default.

File: src/html/translate-html-parser.ts [hunk 1]
```diff
@@ -70,7 +70,7 @@ export  default class TranslateHtmlParser implements htmlparser.Handler {
                 translatedAttributeName = attributeName.substr("translate-attr-".length),
                 defaultText = attributes["translate-default-attr-" + translatedAttributeName];
 
-            this.registerTranslation(translationId, defaultText);
+            this.registerTranslation(translationId, attributes["translate-default"] || defaultText);
```
Comment (DatenMetzgerX) [inline src/html/translate-html-parser.ts]: This is incorrect. If you define one translate-default but the element has multiple translated attributes (translate-attr) elements, then all elements would have the same default text. Therefore, the translate-default-attr-title attribute should be used to define the default text

```html
<a translate translate-attr-title="TITLE-ID" translate-default-attr-title="Default for title attr" href="#"><i class="fa-home fa" /></a>
```

**Comment by pfrankov** [comment]
My fault. Thanks!

**Pull request closed without merging**